docs/getting-started-guides/mesos.md

```diff
@@ -5,20 +5,23 @@
 Mesos allows dynamic sharing of cluster resources between Kubernetes and other first-class Mesos frameworks such as [Hadoop][1], [Spark][2], and [Chronos][3].
 Mesos also ensures applications from different frameworks running on your cluster are isolated and that resources are allocated fairly.
 
-Running Kubernetes on Mesos allows you to easily move Kubernetes workloads from one cloud provider to another to your own physical datacenter.
+Running Kubernetes on Mesos allows you to easily move Kubernetes workloads from one cloud provider to another or to your own physical datacenter.
 
 This tutorial will walk you through setting up Kubernetes on a Mesos cluster.
 It provides a step by step walk through of adding Kubernetes to a Mesos cluster and running the classic GuestBook demo application.
-The walkthrough presented here is based on the v0.4.x series of the Kubernetes-Mesos project, which itself is based on Kubernetes v0.11.0.
+The walkthrough presented here is based on the v0.6.x series of the Kubernetes-Mesos project, which itself is based on Kubernetes v1.0.
```

Comment:
How does a user know what version of the Kubernetes-Mesos project they are using? I think that for this sentence to be useful, you need to tell the user how they can check what Kubernetes-Mesos (and Kubernetes) version they're using.